src/hydro_serving_grpc/writer/writer.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package hydrosphere.writer;
+option java_package = "io.hydrosphere.writer";
+
+import "google/protobuf/empty.proto";
+
+
+message WriteOneRequest {
+    int64 model_version = 1;
+    string feature = 2;
+    bool check = 3;
+    int64 timestamp = 4;
+    int64 uid = 5;
```

Comment:
Does the ID object carry the same key as you provided here with (timestamp, uid) pair? https://github.com/Hydrospheredata/hydro-serving-protos/blob/39e4de50733f0fdc4c6aa1ee46a27827173ff013/src/hydro_serving_grpc/reqstore/reqstore_service.proto#L6

If so, it would be great to reuse the ID message.

Reply:
The contents of this message are still negotiable as table scheme isn't 100% defined yet. But yeah, reusing ID will be handful to increase consistency across services. @mkf-simpson any comments?

Reply:
Oh 😞 There are two objects for request identity: `ID` here and `TraceData` in the monitoring package:
https://github.com/Hydrospheredata/hydro-serving-protos/blob/master/src/hydro_serving_grpc/monitoring/metadata.proto#L12

Reply:
I've substituted `timestamp`, `uid` with `monitoring.TraceData` message.